src/styles/colors.css

```
@@ -10,4 +10,5 @@
   --color-yellow: #ffc700;
   --color-ffffff: #ffffff;
   --color-242a32: #242a32;
+  --color-lightblue-90: #4a90e2;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "--color-lightblue-90" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)